android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -295,22 +296,24 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 				tv.setText(truncateText);
 				tv.setSelection(cursor);
 			}
+		} else if (key.equals(TiC.PROPERTY_BACKGROUND_COLOR)) {
+			tv.setBackgroundColor(Color.TRANSPARENT);
```

Comment:
The below needs to be called after changing TextView background color to transparent since we're in an else-if block (the super class' method only gets called in the else block).

```java
super.propertyChanged(key, oldValue, newValue, proxy);
```